### PR TITLE
Wee bit of maintenance

### PR DESCRIPTION
**Pull request by crazedpsyc**
Added some not-entirely-functional tests and dropped a workaround for an old version of Moo.

### CODE REVIEW

File: lib/App/DuckDuckGo/UI.pm [hunk 1]
```diff
@@ -530,7 +531,7 @@ sub default_bindings {
             my $this = shift;
             # Focus either the ZCI box or the Deep box if this is the last history entry already.
             $zci_box->hidden ? $deep_box->focus : $zci_box->focus, return if $this->history_index >= -1;
-            $this->history_index++;
```
Comment (kentfredric) [inline lib/App/DuckDuckGo/UI.pm]: TBH, I'm surprised that even worked at all to start with. I wasn't aware Moo used lvalue subs.

File: lib/App/DuckDuckGo/UI.pm [hunk 2]
```diff
@@ -671,11 +672,11 @@ sub run {
...
     $self->set_results(
         zci_box => [
-            {'https://duckduckgo.com/'         => '<bold>Homepage</bold>' } ,
-            {'https://duckduckgo.com/about'    => '<bold>About</bold>'    } ,
-            {'https://duckduckgo.com/goodies' => '<bold>Goodies</bold>'  } ,
-            {'https://duckduckgo.com/feedback' => '<bold>Feedback</bold>' } ,
-            {'https://duckduckgo.com/privacy'  => '<bold>Privacy</bold>'  } ,
+            {'https://duckduckgo.com/'         => '<bold>Homepage</bold>' },
+            {'https://duckduckgo.com/about'    => '<bold>About</bold>'    },
+            {'https://duckduckgo.com/goodies'  => '<bold>Goodies</bold>'  },
```
Comment (moollaza) [inline lib/App/DuckDuckGo/UI.pm]: We can actually remove this, the Goodies pages doesn't exist right now

**Comment by moollaza** [comment]
@crazedpsyc this seems to make things work, so that's great.

Right now I can't release the updates though because it looks like the dist.ini needs some updates too:

- The first two are pretty simple and I can fix: 
- `[EOLTest]` => `[Test::EOL]`
- One of `[GitHubMeta]` or `[Repository]` need to be removed because they're both trying to define the same info which throws an error

However the other bug I don't really have the time to investigate, may be a simple fix, but it turns out we were using `[PkgVersion]` and then explicitly defining the version in the code when it wasn't already set. Now PkgVersion skips any file that has `VERSION =` and so we ran into a lot of issues. Are you interested in fixing it for this repo? I'm not familiar with most of this code.

**Comment by crazedpsyc** [comment]
Yeah, never finished the dist.ini originally -- this isn't on cpan. I'll probably fix more in the next few days.

**Comment by moollaza** [comment]
> this isn't on cpan.

Oh my mistake, I assumed it was given the dist.ini and `[@Basic]` -- Well then I guess they're not super important right now :+1:

**Pull request merged**